Sources/Components/Button/MapSettings/MapSettingsButton.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  Copyright © 2019 FINN.no. All rights reserved.
+//
+
+import UIKit
+
+public protocol MapSettingsButtonDelegate: AnyObject {
```

Comment:
You could maybe have one delegate method here and define action enums for each button? Or maybe a better word than action..

`mapSettingsButton(_ mapButton: MapSettingsButton, didSelect action: MapSettingsAction)`